lib/gui/app/pages/main/Flash.tsx

```diff
@@ -219,58 +231,54 @@ export class FlashStep extends React.Component<FlashStepProps, FlashStepState> {
 		});
 	}
 
-	public render() {
-		const state = flashState.getFlashState();
-		const isFlashing = flashState.isFlashing();
-		const flashErrorCode = flashState.getLastFlashErrorCode();
-		return (
-			<>
-				<div className="box text-center">
-					<div className="center-block">
-						<SVGIcon
-							paths={['flash.svg']}
-							disabled={this.props.shouldFlashStepBeDisabled}
+		setErrorMessage(await flashImageToDrive(goToSuccess, source));
```

Comment:
This is broken (probably because of a rebase)

Reply:
Yep I'm fixing it, mainly for the switch from arrow function to class